src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -247,17 +295,19 @@ export default class rd2EntryForm extends LightningElement {
     * and submits them for the record insert or update.
     */
     processSubmit = async (allFields) => {
-        if (this.isElevateWidgetDisplayed) {
+        if (this.isElevateWidgetEnabled) {
             try {
+                this.loadingText = this.customLabels.validatingCardMessage;
                 const elevateWidget = this.template.querySelector('[data-id="elevateWidget"]');
-                const token = await elevateWidget.returnValues().payload;
+                this.paymentMethodToken = await elevateWidget.returnValues().payload;
```

Comment:
I ended with two vars when I expanded this for cardholder name:
- `elevateAuthToken`
- `elevateCardholderName`

Reply:
I will review cardholder name PR this evening, but I would like to keep `paymentMethodToken` name since its aligns with payments services API.